lib/aws/active_job/sqs/cli.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+module Aws
+  module ActiveJob
+    module SQS
+      # Utilities for the aws_active_job_sqs CLI.
+      # @api private
+      module CLI
+        # rubocop:disable Metrics
```

Comment:
You could make individual options into their own methods of you wanted to clean this up more:

```
def queue_option(parser)
  parser.on()...
  end
end
```